assembl/static2/js/app/utils/globalFunctions.js

```diff
@@ -183,48 +184,27 @@ export const isTabletSizedScreen = (screenWidth: number) => screenWidth >= SM_SC
 // works for SCREAMING_SNAKE_CASE, snake_case or cRazY_SNAKE_case
 export const snakeToCamel = (string: string) => string.toLowerCase().replace(/_[a-z]/g, match => match[1].toUpperCase());
 
-// TODO: Replace with js-cookie
-export const getCookieItem = (sKey: string) => {
-  if (!sKey) {
-    return null;
-  }
-  return (
-    decodeURIComponent(
-      document.cookie.replace(
-        new RegExp(`(?:(?:^|.*;)\\s*${encodeURIComponent(sKey).replace(/[-.+*]/g, '\\$&')}\\s*\\=\\s*([^;]*).*$)|^.*$`),
-        '$1'
-      )
-    ) || null
-  );
-};
-
-// TODO: Replace with js-cookie
-const cookie = (key: string, value: string, expires: string, sensitive: boolean = false, path: string = '/'): string => {
-  // As a security policy, we want all cookies that are set to be set according to the scheme
-  // of the URL
-  // In order to test sensitivity, utilize the debateData's requireSecureConnection and what type of
-  // data is being put as a cookie
-  let template = `${key}=${value};path=${path};expires=${expires}`;
-  if (sensitive) {
-    template += ';secure;HttpOnly;';
-  }
-  return template;
-};
+export const getCookieItem = (sKey: string) => Cookies.get(sKey);
 
-// TODO: Replace with js-cookie
 export function setCookieItem(name: string, value: any, sensitive: boolean = false) {
   const date = new Date();
+  let v = null;
```

Comment:
ok so a point - not blocking and certainly not your fault but a variable 'v' - WTF is v, variable names do not cost a thing because they get transpiled anyway, so explain what it's for...ooooohhhhh, so mad...put it in the bin

Reply:
Sigh, changed to `finalValue`